test/TestServerProjects/non-string-enum/Generated/Models/FloatEnum.cs

```diff
@@ -38,6 +38,9 @@ public FloatEnum(float value)
         public static FloatEnum FourHundredSix2 { get; } = new FloatEnum(FourHundredSix2Value);
         /// <summary> 429.1. </summary>
         public static FloatEnum FourHundredTwentyNine1 { get; } = new FloatEnum(FourHundredTwentyNine1Value);
+
+        internal float ToSerialSingle() => _value;
```

Comment:
How do you define the serialization method name? why is the serialization method name of floatEnum `ToSerailSingle`?

Can we just call it `ToSerialize()`?

Reply:
first of all, this is internal, we do not need to care what the name is this.
second, this aligns the behaviour how we get a name of this for fixed enum. We could have fixed enum based on float as well, and there, it is called ToSerialSingle.
The name `ToSerialize` does not make sense. This means to convert this to something else, for instance for int we have `ToSerialInt`.

We get this name using this expression $"ToSerial{ValueType.Name}". In dotnet, float is alias of `System.Single` therefore we see the word single here.

In summary,
1. this aligns with the serialization method name for fixed enum
2. this is internal therefore we do not need really care about this name here.